### PR TITLE
Set tag name format to just be the project version

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Comment by timja** [comment]
Untested

**Comment by jetersen** [comment]
it works pretty well here: https://github.com/jenkinsci/configuration-as-code-plugin/blob/78628c0dfa864c56dacf1a15c8df5a52b70b39fc/pom.xml#L26

File: pom.xml [hunk 1]
```diff
@@ -130,6 +130,12 @@
                     </execution>
                 </executions>
             </plugin>
+          <plugin>
+            <artifactId>maven-release-plugin</artifactId>
+            <configuration combine.children="append">
+              <tagNameFormat>@{project.version}</tagNameFormat>
```
Comment (jetersen) [inline pom.xml:136]: Pretty sure you do not need to go though plugin. Just use the property?

**Review by jetersen — APPROVED** [review]
LGTM

**Review by oleg-nenashev — COMMENTED** [review]
IIRC there was a bug with MRP property resolution at some point, but I do not see why it would not work now. It worth giving it a try

**Pull request merged**